src/server/controller/cycleData/updateOriginalDataPoint.ts

```diff
@@ -16,11 +17,15 @@ export const updateOriginalDataPoint = async (
 ): Promise<OriginalDataPoint> => {
   const { assessment, cycle, originalDataPoint, user } = props
 
+  const updatedOriginalDataPoint = await OriginalDataPointRepository.update(
```

Comment:
why is this outside transaction ?

Reply:
views dont update inside transaction so this needs to be done before, what do you suggest?
(; if we dont move it outside transaction, the updates are 1 state update behind, always showing the previous changed value and calculations)